src/screens/create-wallet-confirm-mnemonic/create-wallet-confirm-mnemonic.tsx

```diff
@@ -160,3 +159,5 @@ export const CreateWalletConfirmMnemonicScreen = smartConnect(
         withTheme(stylesProvider)
     ]
 );
+
+CreateWalletConfirmMnemonicScreenComponent.navigationOptions = navigationOptions;
```

Comment:
i made some changes, i would say fixes in PR #118, so now static properties are moved on the enhanced component by `withTheme` enhancer. add `navigationOptions` on `CreateWalletConfirmMnemonicScreenComponent` before calling `smartConnect`

Reply:
I had no actual changes in this file, some resolve conflicts went horribly wrong.